snap7/server/__main__.py

```diff
@@ -20,18 +21,18 @@
 logger = logging.getLogger("Snap7.Server")
 
 
-@click.command()
-@click.option("-p", "--port", default=1102, help="Port the server will listen on.")
-@click.option(
+@click.command()  # type: ignore[misc]
```

Comment:
whats wrong here that it requires ignoring? maybe add a comment

Reply:
It doesn't matter anymore. I've found a package with stubs for `click`